lib/vehicle.rb

```diff
@@ -0,0 +1,32 @@
+class Vehicle
+  attr_reader :year, :make, :model, :passengers
+  def initialize(year, make, model, passengers = [])
+    @year = year
+    @make = make
+    @model = model
+    @odometer = 0
+    @passengers = []
+  end
+
+  def speeding?
+    if @odometer >= 70
+      true
+    else @odometer < 70
+      false
+    end
+  end
+
+  def speed 
+    @odometer += 70
+  end
+
+  def add_passenger(passenger)
+    @passengers << passenger
+  end
+
+  def num_adults
+    @passengers.count do |passenger|
+      passenger.age >= 18
```

Comment:
```suggestion
      passenger.adult?
```
We can and should use the `adult?` method you wrote in the `Passenger` class here. Lets say someday we decide adults are actually 21 and older, not 18. We would have to change the code anywhere you hard coded 18. If you use the `adult?` method instead, then we only need to change it in the `adult?` method because it will carry through anywhere it is used.